src/main/java/com/revellr/core/partyservice/dto/response/UserResponse.java

```diff
@@ -0,0 +1,24 @@
+package com.revellr.core.partyservice.dto.response;
+
+import lombok.*;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserResponse{
+
+    public String firstName;
+    public String lastName;
+
+    public String getFirstName() {
```

Comment:
no need for getter setters here explicitly. @Getter and @Setter from lombok autogenerates these.